fearless/Modules/BackupWalletImported/BackupWalletImportedInteractor.swift

```diff
@@ -5,7 +5,7 @@ import SSFCloudStorage
 protocol BackupWalletImportedInteractorOutput: AnyObject {}
 
 final class BackupWalletImportedInteractor {
-    var cloudStorageService: FearlessCompatibilityProtocol?
+    var cloudStorageService: CloudStorageServiceProtocol?
```

Comment:
please make sure what CloudStorageServiceProtocol provides right method for fearless compatibility (mobile web) backups